.github/sync.yml

```
@@ -42,6 +42,8 @@ neurobagel/api:
     dest: .github/workflows/build_docker_on_release.yml
     template:
       synced_repo:
+        test_username_unquoted: ${{ secrets.TEST_USERNAME }}
+        test_username_quoted: '${{ secrets.TEST_USERNAME }}'
```

```suggestion
        test_username_unquoted: ${{ secrets.TEST_USERNAME }}
        test_username_quoted: '${{ secrets.TEST_USERNAME }}'
        test_username_github: ${ secrets.TEST_USERNAME }
        test_username_template: {{ secrets.TEST_USERNAME }}
```
can we add two more versions like so: 
1. simple github expansion syntax. Doubt this would trigger either, but we can check
2. that seems to be the syntax on their end. No idea how that would be expanded by Github, but why worry when we can test

---

Good idea!

I'm going to go ahead and merge this with your changes, but I had a question about your suggestion 1.

Not sure if I'm missing something, but based on https://docs.github.com/en/actions/learn-github-actions/variables / https://docs.github.com/en/actions/learn-github-actions/contexts, it seems that the GitHub context expansion syntax always uses two curly braces `${{ }}` rather than one. I guess including `${ secrets.TEST_USERNAME }`, with a single curly brace, could be a way to test for undesirable shell expansion behaviour then? (As I'm not sure this syntax is correct for accessing the `secrets` context).